### PR TITLE
Changed icons8 logo for link icons8

### PR DESCRIPTION
**Pull request by Dajava02**
## Explanation

I've created this pull request to fix this issue https://github.com/Azordev/various-tasks/issues/102

### CODE REVIEW

**Review by Israel-Laguan — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ Happy programming :smiley: :v:
 
 Icons from:
 
-[![Icons8][icons8-logo]][Icons8] Icons8
```
Comment (Katsu08) [inline README.md:125]: @Israel-Laguan the link should go to this page?

image

Comment (Katsu08) [reply README.md:125]: @Dajava02 maybe you could use the Icons8 variable on line 152

https://github.com/Israel-Laguan/parrot-echo/blob/eebb7e843b7f597c7a51e9ab1f7ad08697a9e2a8/README.md?plain=1#L151

You can follow this doc about how to use it https://thehotcode.com/markdown-variables/

Comment (Katsu08) [reply README.md:125]: I think that's fine then. 😁

Comment (Katsu08) [reply README.md:125]: I think that's fine then. 😁

**Review by Katsu08 — APPROVED** [review]
Good work!

**Review by e-azocar — APPROVED** [review]

**Pull request merged**